### ISSUE TITLE
Restrict the use of label and labelledby in sections

### ISSUE THREAD

**Issue by AndersEkl** [nlbdev/nordic-accessible-epub-guidelines#2]
Only major sections should be made landmarks. The use of aria-label and aria-labelledby on section elements must therefore be restricted.

**Comment by indixcss** [comment]
Is there a need to clarify the relationship between the use of landmarks in the two specifications referred to in 2.1 and 2.1.1? One uses aria-labelledby the other epub:type="landmarks".

**Comment by martinpub** [comment]
> I am thinking about how to formulate a simple rule here. Is it a good solution to say that any `<section>` element that has a `role` attribute should also have a `label` or `labelledby` attribute? This would automatically exclude all subsections in chapters. What do you guys say, @martinpub @jonaslil @oscarlcarlsson ?

Good idea, however it may exclude top-level sections where no role is present. Perhaps the rule could be always include for the top level section of each content document?
